frontend/src/components/user/User.tsx

```diff
@@ -1,18 +1,28 @@
-import user from "assets/user.jpg";
+import defaultUserAvatar from "assets/defaultUserAvatar.png";
 import { UserImage, UserName, UserContainer } from "./User.styled";
+import useAvatar from "hooks/useAvatar";
 
 export interface IUser {
-	username: string;
-	id: number;
+    username: string;
+    id: number;
+    userAvatar: {
```

Comment:
You can create IUserAvatar interface for that, that's good practise, it's reusable in the future.